### DISCUSSION TITLE
[Help]: Nested carousels that move along the X-axis

### DISCUSSION THREAD

**Discussion by hustle-dev** [davidjerleke/embla-carousel#959] [Help] [answered]
### Summary

I'm creating a swipeable tabview, each tab has a post with an image slider and I need nested slides. 

Can I use this library for this requirement? 

I've seen some examples, but they don't seem to work properly. There doesn't seem to be any official example yet, so I'm wondering if it would be okay to implement it at the production level using this carousel library.

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment, marked as answer]
Hi @hustle-dev,

Thanks for your question. This is a duplicate of this I think:

- #801 

> I'm wondering if it would be okay to implement it at the production level using this carousel library.

You can achieve it using the [`watchDrag`](https://www.embla-carousel.com/api/options/#watchdrag) option where you would pass a callback to it. There's a link to an unofficial example from an Embla user here:

- https://github.com/davidjerleke/embla-carousel/discussions/801#discussioncomment-8856052 

> There doesn't seem to be any official example yet

Unfortunately no, my time is very limited and this project isn't paying my bills. So the reason there's no official example is because of my lack of spare time and lack of proper financial support for this project (sponsors). 

Best,
David

**Comment by davidjerleke** [comment]
- Duplicate of: #801.
